Common_Voice_Challenge_Terms/en.md

```diff
@@ -0,0 +1,33 @@
+# Voice Pilot Challenge Terms of Service
+Effective November 11, 2019
+
+The Common Voice Pilot Challenge allows you to compete with other people and teams for prizes as you donate to Mozilla’s Common Voice Project. Through Common Voice, you can donate your voice, written sentences, and the other resources we need to build an open-source voice database that anyone can use to make innovative voice recognition apps for devices and the web.
+
+To participate in the Pilot Challenge, you must agree to these terms as well as the [Common Voice Terms of Service](https://voice.mozilla.org/en/terms).
+
+## 1. Eligibility
+
+The Common Voice Pilot Challenge is open to anyone at a participating company who is over the age of 19. Each Employer will have its own additional eligibility requirements.
+
+VOID WHERE PROHIBITED AND WHERE ANY REGISTRATION, BONDING OR LOCALIZATION REQUIRED.
+
+## 2. The Pilot Challenge
+
+Each week during the Pilot Challenge, Mozilla will send you an email announcing the week’s task and how to win. Mozilla will donate $100 to an organization of each individual winner’s choosing (from a list of Mozilla-approved organizations). Mozilla will donate $500 to a charitable organization for each team winner’s choosing (from the same list of Mozilla-approved organizations).
+
+## 3. Communications
+
+When you sign up for the Common Voice Pilot Challenge, you may opt-in to receive emails each week about the Pilot Challenge. If opted-in, you may also receive follow up emails regarding the Pilot Challenge.
+
+## 4. Disclaimers, Governing Law, and Disputes
+
+### Disclaimer, Release and Limit of Liability.
```

Comment:
Nit pick: an extra comma after `Release` to be consistent with the style.

Reply:
Done 👍